docs/sources/reference/commandline/cli.md

```diff
@@ -349,6 +350,31 @@ https://linuxcontainers.org/) via the `lxc` execution driver, however, this is
 not where the primary development of new functionality is taking place.
 Add `-e lxc` to the daemon flags to use the `lxc` execution driver.
 
+### Daemon user namespace support
```

Comment:
I think it's a good idea to clue the reader in as to the reason for the technical limitation. Doing the remapping on a per container basis is probably something a lot of people would want (I know I do) and they're going to end up digging to figure out what's stopping it anyway.